qipr_approver/approver/fixtures/QI_Interest.json

```diff
@@ -0,0 +1,210 @@
+[
+{
+  "model": "approver.qi_interest",
+  "pk": 2,
+  "fields": {
+    "in_registry": false,
+    "created_by": 3,
```

Comment:
please make it so that the person creating these fixtures is the "admin_fixture_user". That should be the user with id=1